force-app/main/default/objects/Tv_Serie__c/validationRules/Tv_Serie_Name_Validation_Rule.validationRule-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Tv_Serie_Name_Validation_Rule</fullName>
+    <active>true</active>
+    <description>Enter the Serie Name, it must contain at least two characters (special characters are not allowed) and should be unique.</description>
```

Comment:
Are you sure that this vr checks if Name is uique?

Reply:
Right, unique is at the second validation rule.